games-simulation/thepowdertoy/thepowdertoy-91.5.330.recipe

```diff
@@ -0,0 +1,71 @@
+SUMMARY="The Powder Toy - A free physics sandbox game"
+DESCRIPTION="The Powder Toy is a free physics sandbox game,\
+which simulates air pressure and velocity, heat, gravity and \
+a countless number of interactions between different substances! \
+The game provides you with various building materials, liquids, gases \
+and electronic components which can be used to construct complex machines,\
+guns, bombs, realistic terrains and almost anything else."
+HOMEPAGE="http://powdertoy.co.uk"
+COPYRIGHT="2008 - 2010 Stanislaw Skowronek.
+	2010 Simon Robertshaw"
+LICENSE="GNU GPL v3"
+REVISION="1"
+SOURCE_URI="https://github.com/simtr/The-Powder-Toy/archive/v$portVersion.tar.gz"
+CHECKSUM_SHA256="9a7de557e771f88e823d06cc21d16f1fbbca930ce751720bc9035d00073d6635"
+SOURCE_DIR="The-Powder-Toy-$portVersion"
+PATCHES="thepowdertoy-$portVersion.patchset"
+ARCHITECTURES="x86_gcc2 x86 x86_64"
+SECONDARY_ARCHITECTURES="x86_gcc2 x86"
```

Comment:
Since it's building for all architectures as primary arch, there's no need for a SECONDARY_ARCHITECTURES. You can remove that and all subsequent $secondaryArchSuffix.